content/en/llm_observability/instrumentation/sdk.md

```diff
@@ -1793,6 +1795,105 @@ def rag_workflow(user_question):
 {{% /tab %}}
 {{< /tabs >}}
 
+## Prompt tracking
+
+Attach structured prompt metadata to the LLM span so you can reproduce results, audit changes, and compare prompt performance across versions. When using templates, LLM Observability also provides [version tracking](#version-tracking) based on template content changes.
+
+### Inline annotation (Python)
+
+Use `LLMObs.annotation_context(prompt=...)` to attach prompt metadata immediately before the LLM call. Use this for one-off or ad‑hoc prompts inside a function. The metadata is added to the current active span (for example, a span from a decorator or an auto-instrumented LLM call). For more details on span annotation, see [Annotating a span](#annotating-a-span).
```

Comment:
I'd go straight to providing an example here rather than describing the usage. This is too many words for a dev which they're not going to read and/or too many words for them to get wrong